dot/package.json

```diff
@@ -2,12 +2,12 @@
   "name": "dot",
   "version": "0.1.0",
   "private": true,
-  "homepage":"https://asierNebiyou.github.io/DOT_DEMO/dot",
+  "homepage": "https://asierNebiyou.github.io/dot_demo2/",
   "dependencies": {
-    "bootstrap": "^5.2.3",
+    "bootstrap": "^5.3.0-alpha1",
```

Comment:
Please avoid `alpha` versions as they are not stable, just use the last stable version.